### PR TITLE
feat : Landing screen completed.

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
Completed the static content for e-com.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
       content="Web site created using create-react-app"
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="stylesheet" href="https://ui-blossom.netlify.app/main-css/main.css">
     <!--
       manifest.json provides metadata used when your web app is installed on a
```
Comment (vidhi-thakur) [inline public/index.html:15]: you can delete the commented code

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
+.item{
+    margin-left: 15px;
```
Comment (vidhi-thakur) [inline src/App.css:2]: try using `rem` for >10px

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+.category-heading{
+    text-align: center;
+}
+.category-div{
+    display: flex;
+    justify-content: space-around;
+}
+.category-img,.category-title-div{
+    height: 250px;
+    width: 250px;
```
Comment (vidhi-thakur) [inline src/App.css:40]: same as above, kindly use `rem`

File: src/App.css [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+}
+.category-title-div{
+    margin-left: -50%;
+    background-color: rgba(0, 0, 0, 0.5);
+}
+.category-title-div:hover{
+    background:none;
+    box-shadow: 5px 5px 5px 5px gray;
```
Comment (vidhi-thakur) [inline src/App.css:48]: you can use the variables you used in your component library here

File: src/App.js [hunk 5]
```diff
@@ -1,11 +1,17 @@
...
+import Footer from "./Components/Home/Footer";
+import Header from "./Components/Home/Header";
+import Landing from "./Components/Home/Landing";
 
 function App() {
+  
   return (
     <div className="App">
-      
+      <Header />
```
Comment (vidhi-thakur) [inline src/App.js:10]: you can install `react-router-dom` soon and after every page you make, just add the route

File: src/Components/Home/Footer.jsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+            
+               <ul className="quick-link-list">
+                  <li className="quick-link-item"> About Us </li>
+                  <li className="quick-link-item"> Terms & Conditions </li>
+                  <li className="quick-link-item"> Privacy policy </li>
+               </ul>           
+               <ul className="social-media">
+                  <li className="social-media-item"> <a href="/"> Github </a> </li>
+                  <li className="social-media-item"> <a href="/"> Twitter </a> </li>
+                  <li className="social-media-item">  <a href="/"> Instagram</a> </li>
```
Comment (vidhi-thakur) [inline src/Components/Home/Footer.jsx:14]: Please add the links to your profile

File: src/Components/Home/Header.jsx [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+       <li className="list-item">
+        <button className="secondary-btn btn">Sign Up</button>
+       </li>
+       <li className="list-item flex">
+         <a className="item" href="/">
+           <FaShoppingCart />
+         </a>
+       </li>
+       <li className="flex">
+          <a className="item" href="/">
```
Comment (vidhi-thakur) [inline src/Components/Home/Header.jsx:28]: can use `Link` here after configuring react router

**Review by vidhi-thakur — COMMENTED** [review]
Added few comments, please check.

**Comment by vermasavita** [comment]
Make the changes @vidhi-thakur suggested.

File: src/App.js (hunk 5 above)
Comment (shuklahimanshu81) [reply src/App.js:10]: will be change them after all the static content is done

**Review by vermasavita — APPROVED** [review]

File: src/Components/Home/Header.jsx (hunk 7 above)
Comment (shuklahimanshu81) [reply src/Components/Home/Header.jsx:28]: Will do that in upcoming PRs

**Pull request merged**